src/utils/format_result.ts

```diff
@@ -15,6 +16,7 @@ export const formatZipResult = (archive: FunctionArchive) => {
     inSourceConfig: undefined,
     runtime: archive.runtime.name,
     schedule: archive.inSourceConfig?.schedule ?? archive?.config?.schedule,
+    runtimeAPIVersion: archive.inSourceConfig?.runtimeAPIVersion,
```

Comment:
Should there also be a fallback to `1` here?
```suggestion
    runtimeAPIVersion: archive.inSourceConfig?.runtimeAPIVersion ?? 1,
```

Reply:
No, the decision if it is 1 or 2 happens during parsing of the file. At this point if it is undefined it means we do not know the version, because of parsing errors or other things gone wrong. And i do not want to return a version that might be wrong.